dGame/dBehaviors/BehaviorSlot.h

```diff
@@ -1,5 +1,8 @@
 ﻿#pragma once
 
+#ifndef BEHAVIORSLOT_H
```

Comment:
What's with the traditional header guards in addition to `#pragma once`?

Reply:
For some reason the pragma once is not preventing the file from being included twice in a compile, but using traditional header guards prevented the issue
![image](https://user-images.githubusercontent.com/39972741/174507208-d2af0d65-613e-4db0-9d1f-cb7a53fa2aaf.png)

Do you know of a way I can use the PCH while still using pragma once?

Reply:
I mean `#pragma once` is non-standard (but it's implemented by *most* compilers). I've advocated in the past for traditional guards to be used for this very reason. I don't know of any way to fix this, what compiler are you using?

Reply:
For now ill be only using the Entity.h PCH since thats the only one that doesnt require us to add the traditional include guards.  Will look into a way to make sure this compiles on gcc with other PCH in the future.  Even for now, having that one file as a PCH has reduced compile times drastically

Reply:
There may be other third party headers I can pre compile I that ill look into though